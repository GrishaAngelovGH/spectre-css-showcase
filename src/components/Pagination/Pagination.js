const Pagination = () => {
  return (
    <ul className='pagination'>
      <li className='page-item disabled'>
        <a href='#' tabIndex='-1'>Previous</a>
      </li>
      <li className='page-item active'>
        <a href='#'>1</a>
      </li>
      <li className='page-item'>
        <a href='#'>2</a>
      </li>
      <li className='page-item'>
        <a href='#'>3</a>
      </li>
      <li className='page-item'>
        <span>...</span>
      </li>
      <li className='page-item'>
        <a href='#'>12</a>
      </li>
      <li className='page-item'>
        <a href='#'>Next</a>
      </li>
    </ul>
  )
}

export default Pagination