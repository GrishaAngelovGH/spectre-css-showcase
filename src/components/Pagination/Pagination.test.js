import { render } from '@testing-library/react'

import Pagination from './Pagination'

test('should render Pagination component', () => {
  const view = render(<Pagination />)

  expect(view).toMatchSnapshot()
})