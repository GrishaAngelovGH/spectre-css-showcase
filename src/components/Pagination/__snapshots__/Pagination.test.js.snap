// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Pagination component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <ul
        class="pagination"
      >
        <li
          class="page-item disabled"
        >
          <a
            href="#"
            tabindex="-1"
          >
            Previous
          </a>
        </li>
        <li
          class="page-item active"
        >
          <a
            href="#"
          >
            1
          </a>
        </li>
        <li
          class="page-item"
        >
          <a
            href="#"
          >
            2
          </a>
        </li>
        <li
          class="page-item"
        >
          <a
            href="#"
          >
            3
          </a>
        </li>
        <li
          class="page-item"
        >
          <span>
            ...
          </span>
        </li>
        <li
          class="page-item"
        >
          <a
            href="#"
          >
            12
          </a>
        </li>
        <li
          class="page-item"
        >
          <a
            href="#"
          >
            Next
          </a>
        </li>
      </ul>
    </div>
  </body>,
  "container": <div>
    <ul
      class="pagination"
    >
      <li
        class="page-item disabled"
      >
        <a
          href="#"
          tabindex="-1"
        >
          Previous
        </a>
      </li>
      <li
        class="page-item active"
      >
        <a
          href="#"
        >
          1
        </a>
      </li>
      <li
        class="page-item"
      >
        <a
          href="#"
        >
          2
        </a>
      </li>
      <li
        class="page-item"
      >
        <a
          href="#"
        >
          3
        </a>
      </li>
      <li
        class="page-item"
      >
        <span>
          ...
        </span>
      </li>
      <li
        class="page-item"
      >
        <a
          href="#"
        >
          12
        </a>
      </li>
      <li
        class="page-item"
      >
        <a
          href="#"
        >
          Next
        </a>
      </li>
    </ul>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
